src_ts/components/pages/interventions/list/filters.ts

```diff
@@ -3,7 +3,9 @@ import {AnyObject} from '../../../../types/globals';
 import {isJsonStrMatch} from '../../../utils/utils';
 import {RouteQueryParams} from '../../../../routing/router';
 
-export const DRAFT_STATUS = 'draft';
+export const DEVELOPMENT_STATUS = 'development';
```

Comment:
on bk it remains as 'draft' , on FE we show it as 'Development'
```suggestion
export const DEVELOPMENT_STATUS = 'draft';
```